src/main/scala/org/constellation/consensus/Consensus.scala

```diff
@@ -481,27 +523,38 @@ object Consensus {
     facilitatorId: FacilitatorId,
     transactions: List[Transaction],
     tipsSOE: TipSoe,
-    messages: Seq[ChannelMessage]
+    messages: Seq[ChannelMessage],
+    observations: List[Observation]
   )
 
   case class StopBlockCreationRound(
     roundId: RoundId,
     maybeCB: Option[CheckpointBlock],
-    transactionsToReturn: Seq[String]
+    transactionsToReturn: Seq[String],
+    observationsToReturn: Seq[String]
   ) extends RoundCommand
 
-  case class EmptyProposals(roundId: RoundId, stage: String, transactionsToReturn: Seq[String])
-      extends ConsensusException(s"Proposals for stage: $stage and round: $roundId are empty.")
+  case class EmptyProposals(
+    roundId: RoundId,
+    stage: String,
+    transactionsToReturn: Seq[String],
+    observationsToReturn: Seq[String]
```

Comment:
Do we need this two variables here "transactionsToReturn" and "observationsToReturn" 
they are not used in any error message

Reply:
Yes, both are actually used in the code:
```
def handleRoundError(cmd: ConsensusException): F[Unit] =
    for {
      _ <- logger.error(cmd)(s"Consensus with roundId: ${cmd.roundId} finished with error: ${cmd.getMessage}")
      _ <- stopBlockCreationRound(
        StopBlockCreationRound(cmd.roundId, None, cmd.transactionsToReturn, cmd.observationsToReturn)
      )
    } yield ()
```